### PR TITLE
Add Safari 11

### PR DESCRIPTION
**Pull request by chicoxyzzy**

### CODE REVIEW

**Review by ljharb — COMMENTED** [review]
How are you installing Safari 11 to test this? I don't have it available on my Mac OS app update; it's not on browserstack; iOS hasn't released with it yet.

**Comment by chicoxyzzy** [comment]
@ljharb I have Apple Developer Account. It's developer betas. I have both MacOS High Sierra and iOS 11 installed on my devices.

File: environments.json [hunk 1]
```diff
@@ -1381,6 +1381,13 @@
     "release": "2017-03-27",
     "obsolete": false
   },
+  "safari11": {
+    "full": "Safari 11",
+    "family": "JavaScriptCore",
+    "short": "SF 11",
+    "release": "2017-06-05",
+    "unstable": false
```
Comment (afmenez) [inline environments.json]: Was it released already?  I was under the impression that it is still in beta...

File: environments.json [hunk 2]
```diff
@@ -1717,5 +1724,14 @@
...
     "release": "2017-03-27",
     "obsolete": false
+  },
+  "ios11": {
+    "full": "iOS Safari",
+    "family": "iOS Safari",
+    "short": "iOS 11",
+    "platformtype": "mobile",
+    "equals": "safari11",
+    "release": "2017-06-05",
```
Comment (afmenez) [inline environments.json]: I believe it was not released yet, so this date seems wrong...

**Review by afmenez — CHANGES_REQUESTED** [review]
Almost there.

**Review by afmenez — APPROVED** [review]
LGTM

File: environments.json (hunk 1 above)
Comment (ljharb) [reply environments.json]: good call; beta should be considered unstable.

Comment (chicoxyzzy) [reply environments.json]: Yes, I fixed that. Thank you @afmenez

**Pull request merged**